inspire/modules/authors/recordext/fields/author.cfg

```diff
@@ -0,0 +1,400 @@
+                        'status': value['z']}
+    producer:
+        json_for_marc(), {"371__a": "institution", "371__r": "rank",
+                          "371__m": "email", "371__o": "old_email",
+                          "371__s": "start_date", "371__t": "end_date",
+                          "371__z": "status"}
+
+
+other_names:
+    """Contains other variation of names.
+
+    Usually a different form of writing the primary name.
+
+    TO DO: define a type inside list"""
+    schema:
+        {'other_names': {'type': 'list', 'force': True}}
+    creator:
+        marc, "400__", value['a']
+    producer:
+        json_for_marc(), {"400__", ""}
+
+previous_names:
+    """Contains a previous name.
+
+    For example, it can be a maiden's name.
+
+    TO DO: define a type inside list"""
+    schema:
+        {'previous_name': {'type': 'list', 'force': True}}
+    producer:
+        json_for_marc(), {"400__a": "", "400__w": "a"}
+
+private:
+    """Contains hidden information.
+
+    Example
+
+    +---------+---------------------+
+    | 595__a  | I like bananas      |
+    +---------+---------------------+
+    | 595__m  | pony@cern.ch        |
+    +---------+---------------------+
+    | 595__o  | serious@ema.il      |
+    +---------+---------------------+
+    """
+    schema:
+        {'private': {'type': 'dict', 'schema': {
+         'current_mail': {'type': 'string', 'empty': False, 'nullable': True,
+                          'regex': __import__('flask').current_app.config['INSPIRE_EMAIL_REGEX']},
+         'old_mail': {'type': 'string', 'empty': False, 'nullable': True,
+                      'regex': __import__('flask').current_app.config['INSPIRE_EMAIL_REGEX']},
+         'note': {'type': 'string', 'empty': False, 'nullable': True}
+         }}}
+    creator:
+        marc, "595__", {'note': value['a'], 'current_mail': value['m'],
+                        'old_mail': value['o']}
+    producer:
+        json_for_marc(), {"595__a": "note", "595__m": "current_mail",
+                          "595__o": "old_mail"}
+
+field_categories:
+    """Contains field category.
+
+    Example:
+
+    +---------+---------------------+
+    | 65017a  | comp                |
+    +---------+---------------------+
+    | 650172  | arxiv               |
+    +---------+---------------------+
+
+    In JSON stored as a list of dictionaries.
+    """
+    schema:
+        {'field_categories': {'type': 'list', 'force': True, 'items': {
+         'name': {'type': 'string',
+                  'allowed': __import__('flask').current_app.config['INSPIRE_ARXIV_CATEGORIES']},
+         'type': {'type': 'string',
+                  'allowed': __import__('flask').current_app.config['INSPIRE_CATEGORIES_SOURCES']}
+         }}}
+    creator:
+        marc, "65017", {'name': value['a'], 'type': value['2']}
+    producer:
+        json_for_marc(), {"65017a": "name", "650172": "type"}
+
+curators_note:
+    """Contains curators note."""
```

Comment:
This is not viewable by the user. I won't answer the second question - you should discuss it with Anette.